src/app/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import OverlayWrapper from '../OverlayWrapper/OverlayWrapper'
+import { FaEllipsisH } from 'react-icons/fa'
+
+export default function DropdownMenu({
+  children,
+}: React.PropsWithChildren<React.ReactFragment>): JSX.Element {
+  const [showMenu, setShowMenu] = useState(false)
+  return (
+    <>
+      <MenuButton onClick={() => setShowMenu(true)} />
+      {showMenu && (
```

Comment:
for me it would be bette for understanding, if you put it in an if statement. I had to read 5 times to get whats goin on :)